aries-site/src/examples/cardPreviews/toast-notification.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { StatusGoodSmall } from 'grommet-icons/icons/StatusGoodSmall';
+import { FormClose } from 'grommet-icons/icons/FormClose';
+
+import { Box, Button, Text } from 'grommet';
+
+export const ToastPreview = () => (
+    <Box align="center" justify="center" fill background="background-back">
+      <Box 
+            elevation="medium" 
+            round="xsmall" 
+            pad={{ horizontal: 'small', vertical: 'xsmall' }} 
+            background={{ color: 'background-front' }}
+      >
+        <Box direction="row">
+          <Box pad={{ right: 'small' }}>
+            <StatusGoodSmall color="status-ok" />
+          </Box>
+          <Box
+            gap="medium"
+            align="start"
+            direction="row"
+            justify="between"
+            fill
+          >
+            <Box>
+                <Text 
+                    weight="bold" 
+                    color={{ light: 'black', dark: 'white' }}
+                >Hooray!</Text>
+                <Text 
+                    color={{ light: 'black', dark: 'white' }}
+                >Your toast is done!</Text>
+            </Box>
+            <Button
+              icon={<FormClose color={{ light: 'black', dark: 'white' }} />}
+              plain
+            />
```

Comment:
Eliminate double focus / button within a button:
![Screen Shot 2021-08-13 at 2 59 21 PM](https://user-images.githubusercontent.com/1756948/129418006-b4a84e65-ce44-494b-a8a8-3320d3ea3cd9.png)


```suggestion
            <Button
              icon={<FormClose color={{ light: 'black', dark: 'white' }} />}
              plain
              tabIndex={-1}
            />
```